gulpfile.js

```diff
@@ -140,7 +140,11 @@ gulp.task('vsce-publish', function () {
     return vsce.publish();
 });
 gulp.task('vsce-package', function () {
-    return vsce.createVSIX();
+    const usePackagePathOptionIndex = process.argv.findIndex(arg => arg === "--packagePath");
+    const options = (usePackagePathOptionIndex >= 0 && usePackagePathOptionIndex < process.argv.length)
+        ? { packagePath: process.argv[usePackagePathOptionIndex + 1] }
```

Comment:
Care extracting this `process.argv[usePackagePathOptionIndex + 1]` as a separate variable? Will improve readability.

Reply:
That is a good idea